src/crustacean/core.clj

```diff
@@ -167,19 +186,19 @@
 
 (defn ->malformed?*
   "The malformed? database function for a given entity"
```

Comment:
This docstring is sad panda :P. What does it doooooooooooo?

(Also: valid? probs better n'est pas)